next.config.js

```diff
@@ -55,6 +55,9 @@ plugins.push(withTM);
 
 // prettier-ignore
 module.exports = () => plugins.reduce((acc, next) => next(acc), {
+  env: {
+    BLOXY_API_KEY: process.env.BLOXY_API_KEY
+  },
```

Comment:
If we're are going to use this on the client we should rename this to `process.env.NEXT_PUBLIC_BLOXY_API_KEY` and use it directly where needed.

```suggestion
```